### DISCUSSION TITLE
Calling softare update via command?

### DISCUSSION THREAD

**Discussion by cooka90** [cockpit-project/cockpit#16449] [Q&A] [answered]
OS: Ubuntu
Page: Software updates

Updating server via software updates through terminal.

Hi All,

The software update tab on cockput is really useful, especially to see update history and available updates etc. Is it possible, to call software update without going through the gui (in terminal), but when the gui is opened the update history is still displayed?

Many thanks

**Comment by martinpitt** [comment, marked as answer]
Yes, see https://cockpit-project.org/guide/latest/feature-packagekit.html. In short, use `pkcon update`, a CLI which uses PackageKit.
